build/release.mk

```diff
@@ -159,13 +159,14 @@ else
 	cp $(GOBIN)/$(CURRENT_PACKAGE_ARCH)/$(MM_BIN_NAME) $(DIST_PATH_GENERIC)/bin # from cross-compiled bin dir
 endif
 
+	#Download MMCTL for $(MMCTL_PLATFORM)
+	scripts/download_mmctl_release.sh $(MMCTL_PLATFORM) $(DIST_PATH_GENERIC)/bin
```

Comment:
I'm not sure I understand, this script only downloads the `amd64` version of mmctl, but we want the arm64 version ?

Reply:
@ashishbhate we are working to start releasing linux arm64 packages as well, so this is needed